TheMET/TheMET/ViewControllers/CatalogViewController.swift

```diff
@@ -86,19 +86,29 @@ class CatalogViewController: UIViewController {
                 completion(nil)
                 return
             }
-            self?.loadCatalogCellDataList(departments: responce.departments) { catalogCellDataList in
+            var catalogCellDataList: [CatalogCellData] = []
+       for department in responce.departments {
+            let catalogCellData: CatalogCellData = CatalogCellData(departmentId: department.id, departmentData: .placeholder)
+           catalogCellDataList.append(catalogCellData)
+        }
+            self?.contentStatus = .loaded(catalogCellDataList)
+            self?.loadCatalogCellDataList(departments: responce.departments, catalogCellDataList: catalogCellDataList) { catalogCellDataList in
                 completion(catalogCellDataList)
             }
         }
     }
     
-    private func loadCatalogCellDataList(departments: [Department], completion: @escaping ([CatalogCellData]) -> Void) {
+    private func loadCatalogCellDataList(departments: [Department], catalogCellDataList: [CatalogCellData], completion: @escaping ([CatalogCellData]) -> Void) {
```

Comment:
This method also interacts with `contentStatus` on its own. 
- `completion` is not needed anymore
- rename method to `reloadCatalog(departments:, catalogCellDataList:)`

Reply:
+